troi/patches/daily_jams.py

```diff
@@ -23,11 +23,25 @@ class DailyJamsPatch(troi.patch.Patch):
     """
 
     @staticmethod
-    @cli.command(no_args_is_help=True)
-    @click.argument('user_name')
-    @click.argument('jam_date', required=False)
-    def parse_args(**kwargs):
-        """
+    def get_args():
+        return [
+            {
+                "type": "argument",
```

Comment:
To get a better idea as to where this is going, it would be nice to convert a patch that has more args, or perhaps document these dicts a better... I hate that we're having to duplicate such an important module such as click, but that seems to be our lot in this life...

Overall, I think this will be fine for what we need -- it seems that there is enough flexibility to add support for future more complicated command line args as we go.